### ISSUE TITLE
dead Northeast Scala Symposium links

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-lang#1491]
htmlproofer says:

```
* At ./_site/pastevents/index.html:280:

  External link https://nescala.io/2011/index.html failed (status code 410)

* At ./_site/pastevents/index.html:4[13](https://github.com/scala/scala-lang/actions/runs/4727082505/jobs/8387302448#step:6:14):

  External link https://nescala.io/2012/index.html failed (status code 410)

* At ./_site/pastevents/index.html:557:

  External link https://nescala.io/2013/index.html failed (status code 410)

* At ./_site/pastevents/index.html:3084:

  External link https://nescala.io/20[17](https://github.com/scala/scala-lang/actions/runs/4727082505/jobs/8387302448#step:6:18)/index.html failed (status code 410)

* At ./_site/pastevents/index.html:3802:

  External link https://nescala.io/20[18](https://github.com/scala/scala-lang/actions/runs/4727082505/jobs/8387302448#step:6:19)/index.html failed (status code 410)
```
 
"[The domain name nescala.io is for sale](https://nescala.io/)"

fyi @jducoeur

**Comment by jducoeur** [comment]
Yeah, this one's a pure mea culpa - before I got my act together and started to actually organize 2023, a domain squatter got to the domain, somewhere around 4-6 months ago.  My current plan is to set up a redirect under sca.la as the hopefully-permanent address, and then we can figure out what that really points to.

Hadn't realized there were historical links from here to older NE Scalas; we'll have to figure out how to stand those up somewhere else.  I've added a matching issue in the NE Scala tracker: https://github.com/nescalas/nescala-2023-planning/issues/12

Thanks for calling this to my attention!
